app/admin/top-issues/components/AdminTopIssuesPage.js

```diff
@@ -1,32 +1,35 @@
 'use client';
 import PortalPanel from '@shared/layouts/PortalPanel';
 import AdminWrapper from 'app/admin/shared/AdminWrapper';
-import Tooltip from '@mui/material/Tooltip';
-import Table from '@shared/utils/Table';
-import { useMemo, useState } from 'react';
-import { formatToPhone } from 'helpers/numberHelper';
-import gpApi, { isProd } from 'gpApi';
+import React, { useState } from 'react';
+import gpApi from 'gpApi';
 import gpFetch from 'gpApi/gpFetch';
-import AlertDialog from '@shared/utils/AlertDialog';
-import { FaTrash } from 'react-icons/fa';
-
 import { useHookstate } from '@hookstate/core';
 import { globalSnackbarState } from '@shared/utils/Snackbar';
 import BlackButtonClient from '@shared/buttons/BlackButtonClient';
 import TextField from '@shared/inputs/TextField';
 import TopIssuesList from './TopIssuesList';
+import { SVGIconChooser } from '@shared/buttons/SVGIconChooser';
```

Comment:
Does this deserve to be in the shared folder? What else is sharing it?

Reply:
It's being used in two different files.

<img width="696" alt="image" src="https://github.com/thegoodparty/gp-webapp/assets/496983/98be867e-a6cc-4e87-b3da-0f311b445f92">

I'm fine w/ moving it wherever you'd like me to.

Reply:
Both of those files are under admin/top-issues
You can either create a shared folder there or add it to one of the components' folders under it. 
the main shared folder are reserved for global elements. If something is shared just for the onboarding for example, it belongs to /onboarding/shared, not the main shared.